torchvision/ops/roi_align.py

```diff
@@ -35,6 +35,15 @@ def roi_align(input, boxes, output_size, spatial_scale=1.0, sampling_ratio=-1, a
     Returns:
         output (Tensor[K, C, output_size[0], output_size[1]])
     """
+    if isinstance(boxes, list):
+        for _tensor in boxes:
+            assert _tensor.size(1) == 4, \
+                'The shape of the tensor in the boxes list is not correct as List[Tensor[L, 4]]'
+    elif isinstance(boxes, torch.Tensor):
+        assert boxes.size(1) == 5, 'The boxes tensor shape is not correct as Tensor[K, 5]'
+    else:
+        assert False, 'boxes shape is not correct'
```

Comment:
Looks like this block of code is the same for both `roi_align` and `roi_pool` (and we should also add it for `ps_roi_align` and `ps_roi_pool`. Can you factor it out into a helper function (maybe called `check_boxes_is_valid`?) and put it in `_utils.py`, like we did for `convert_boxes_to_roi_format`?